### PR TITLE
Pass through `crs` when using `points_from_xy`

### PR DESCRIPTION
**Pull request by gadomski**
This enables the usage [in the readme](https://github.com/geopandas/dask-geopandas/blob/master/README.rst#example) to have a CRS on the resultant data frame (which is needed for writing to parquet).

Sidecar fixes:
- Swaps 'longitude' and 'latitude' in the readme as they were incorrect
- Adds a `requirements-dev.txt` file that contains the packages needed to run the test suite locally

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -60,7 +60,7 @@ data.
 
    ddf = dask_geopandas.from_dask_dataframe(ddf)
    ddf = ddf.set_geometry(
-       dask_geopandas.points_from_xy(ddf, 'latitude', 'longitude')
+       dask_geopandas.points_from_xy(ddf, 'longitude', 'latitude')
```
Comment (jorisvandenbossche) [inline README.rst:63]: Whoops .. thanks for the catch!

File: tests/test_core.py [hunk 2]
```diff
@@ -135,6 +135,19 @@ def test_points_from_xy():
...
+    latitude = [40.2, 66.3]
+    longitude = [-105.1, -38.2]
+    expected = geopandas.points_from_xy(longitude, latitude, crs="EPSG:4326")
+    df = pd.DataFrame({"longitude": longitude, "latitude": latitude})
+    ddf = dd.from_pandas(df, npartitions=2)
+    actual = dask_geopandas.points_from_xy(
+        ddf, "longitude", "latitude", crs="EPSG:4326"
+    )
+    assert isinstance(actual, dask_geopandas.GeoSeries)
+    assert list(actual) == list(expected)
```
Comment (jorisvandenbossche) [inline tests/test_core.py:148]: This won't in itself verify the that the crs is set, so maybe you can change this with `assert_geoseries_equal(actual.compute(), expected)`

**Review by jorisvandenbossche — COMMENTED** [review]
Thanks for the PR! One small comment on the test

File: tests/test_core.py (hunk 2 above)
Comment (gadomski) [reply tests/test_core.py]: Updated in 88f03cf.

**Review by jorisvandenbossche — APPROVED** [review]

**Review by martinfleis — APPROVED** [review]
Thanks!

**Pull request merged**